xml/rmt_client.xml

```diff
@@ -228,6 +228,37 @@
     the module.
    </para>
  </sect1>
+ <sect1 xml:id="rmt-client-custom-repo">
+  <title>Configuring Clients for Custom Stand-alone Repositories</title>
+  <para>
+   If you created a custom stand-alone repository on the &rmt; server, it will
+   not be registered on client machines with <command>SUSEConnect</command>
+   because it has no parent product.
+  </para>
+  <para>
+   To add the repository manually, follow these steps:
+  </para>
+  <procedure>
+   <step>
+    <para>
+     Point your Web browser to the following RMT server URL:
+    </para>
+<screen>https://<replaceable>RMT_SERVER_HOSTNAME/repo/</replaceable></screen>
+   </step>
+   <step>
+    <para>
+     Navigate the browser through the directory structure to your custom
+     repository's <filename>repodata/</filename> subdirectory.
+    </para>
+   </step>
+   <step>
+    <para>
+     On the client machine, add the discovered repository URL:
+    </para>
+<screen>&prompt.sudo;zypper ar <replaceable>CUSTOM_REPO_URL</replaceable></screen>
```

Comment:
This command is missing the repo name. The call should look like: 

`zypper ar <URI> <Alias>`